apps/messages/lib.js

```diff
@@ -102,7 +102,14 @@ exports.clearAll = function(event) {
   if (inApp) return onMessagesModified();
   // if we have a widget, update it
   if (global.WIDGETS && WIDGETS.messages)
-    WIDGETS.messages.hide();
+    WIDGETS.messages.update(messages);
+}
+
+function getNotificationImage() {
```

Comment:
Thanks! However are `getNotificationImage` and `getFBIcon` needed now? I think they might be left over from before the most recent changes

Reply:
Fixed in [f3d3091](https://github.com/espruino/BangleApps/pull/1872/commits/f3d309170675929ec667f0d44dba88e3eea83028)